app/controllers/sections_controller.rb

```diff
@@ -27,8 +27,9 @@ def show_section
   def show_events
     subsection_events = @section.subsections.find_by(name: 'events')
     @events = subsection_events.categories.find_by(name: 'Eventos')
-    @highlight = @events.videos.find_by(highlighted: 'true') ||
-                 @events.series.find_by(highlighted: 'true')
+    @highlight =
```

Comment:
Just as a comment, you could have created `videos` and `series` variables and used `@highlight = videos || series` ;)